src/ui/src/shape_list_widget.cpp

```diff
@@ -1,10 +1,172 @@
 #include "shape_list_widget.h"
 
-#include "ui_shape_list_widget.h"
+#include <QStringList>
+#include <QFile>
+
 
 ShapeListWidget::ShapeListWidget(QWidget* parent)
-		: QDockWidget(parent), ui_(new Ui::ShapeListWidget) {
-	ui_->setupUi(this);
+		: QDockWidget(parent) {
+
+	// example with checkboxes https://stackoverflow.com/questions/14158191/qt-qtreeview-and-custom-model-with-checkbox-columns
+	
+	model_ = new TreeModel(this);
+	view_ = new QTreeView(this);
+	view_->setModel(model_);
+	view_->setWindowTitle(QObject::tr("Simple Tree Model"));
+	setWidget(view_);
+
+	Test();
+}
+
+void ShapeListWidget::Test() {
+	model_->TestFillWithTxtFile(
+			"C:\\Users\\winroot\\Desktop\\overheat\\local_docs\\default.txt");
```

Comment:
Will be work only on Peter's windows machine with concrete username. Remove it (will be work with directory of executable easier with global config, another reminder for me)